doc/source/ray-core/api/utility.rst

```diff
@@ -10,6 +10,8 @@ Utility
    ray.nodes
    ray.cluster_resources
    ray.available_resources
+   ray.util.queue.Empty
```

Comment:
can you comment inline why we're adding these to the doc, thankks

Reply:
Sure, but it's because they're referred to elsewhere in the documentation.